server/src/main/java/org/candlepin/model/Consumer.java

```diff
@@ -124,15 +124,30 @@
     @Size(max = 255)
     private String serviceLevel;
 
-    @Column(length = 255, nullable = true)
+    @Column(name = "sp_role", length = 255, nullable = true)
     @Size(max = 255)
-    private String offering;
+    private String role;
 
-    //usage is a key word for mysql
-    @Column(length = 255, nullable = true, name = "intent_usage")
+    @Column(name = "sp_usage", length = 255, nullable = true)
     @Size(max = 255)
     private String usage;
 
+    @ElementCollection
+    @CollectionTable(name = "cp_sp_add_on", joinColumns = @JoinColumn(name = "consumer_id"))
+    @Column(name = "add_on")
+    private Set<String> addOns = new HashSet<>();
+
+    @Column(name = "sp_status", length = 32)
+    @Size(max = 32)
+    private String systemPurposeStatus;
+
+    /**
+     * Represents an SHA256 hex string of the last calculated system purpose status.
```

Comment:
It's murmur3 now that #2051 is merged.  I would just say "a 256 bit hash digest" to keep it vague.

Reply:
@awood Is the compliance status now murmur3 as well? If so, I'll update the comment for complianceStatusHash at the same time.

Reply:
Looks like it is. I'll change the comment there as well.

Reply:
Done. Will be included in next update of this PR. Thanks!